frontend/views/containers/payments/PaymentRowSendLightning.vue

```diff
@@ -0,0 +1,132 @@
+<template lang="pug">
+  // Note: .cpr- is for payment-row
+  payment-row(:payment='payment')
+    template(slot='cellPrefix')
+      label.checkbox.c-check(data-set='check')
+        input.input(type='checkbox' v-model='form.checked')
+        span
+          i18n.sr-only Mark as an item to pay
+
+    template(slot='cellUser')
+      .c-user-wrapper
+        .c-user-avatar-name
+          avatar-user.c-avatar(:username='payment.username' size='xs')
+          strong.c-name {{ payment.displayName }}
+        i18n.pill.is-neutral.hide-tablet(tag='div') Lightning
+
+    template(slot='cellActions')
+      i18n.pill.is-neutral.hide-phone Lightning
+      i18n.is-unstyled.link.is-link-inherit.c-reset(
+        tag='button'
+        type='button'
+        v-if='payment.amount !== config.initialAmount'
+        @click='reset'
+      ) Reset
+
+    template(slot='cellSuffix')
+      label.field
+        .inputgroup
+          input.input(data-test='amount' inputmode='decimal' pattern='[0-9]*' v-model='form.amount')
+          .suffix.hide-phone {{ currencies.symbolWithCode }}
+          .suffix.hide-tablet {{ currencies.symbol }}
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import currencies from '@model/contracts/shared/currencies.js'
+import PaymentRow from './payment-row/PaymentRow.vue'
+import AvatarUser from '@components/AvatarUser.vue'
+
+export default ({
+  name: 'PaymentRowSendLightning',
+  components: {
+    PaymentRow,
+    AvatarUser
+  },
+  props: {
+    payment: {
+      type: Object, // { index, checked, ...paymentsData }
+      required: true
+    }
+  },
+  data () {
+    const gCurrency = currencies[this.$store.getters.groupSettings.mincomeCurrency]
+    return {
+      config: {
+        initialAmount: this.payment.amount
+      },
+      form: {
+        checked: this.payment.checked,
+        amount: gCurrency.displayWithoutCurrency(this.payment.amount)
+      }
+    }
+  },
+  watch: {
+    'form.amount' (amount) {
+      this.$emit('update', {
+        index: this.payment.index,
+        checked: true,
+        amount
+      })
+    },
+    'form.checked' (checked) {
+      this.$emit('update', {
+        index: this.payment.index,
+        checked
+      })
+    },
+    'payment.checked' (checked) {
+      this.form.checked = checked
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'groupSettings'
+    ]),
+    currencies () {
+      return currencies[this.groupSettings.mincomeCurrency]
+    }
```

Comment:
FYI this already exists as a getter that you can import. It's called `groupCurrency` and it's defined in the group contract. All of those getters are imported into the Vuex getters in `state.js`, so you should be able to use `mapGetters` to get it here instead of redefining it.